src/App.vue

```diff
@@ -889,28 +696,6 @@ export default class App extends Mixins(
       await this.fetchData()
     }
   }
-
-  /** Request and Download Business Summary Document. */
-  async downloadBusinessSummary (): Promise<void> {
-    this.setFetchingDataSpinner(true)
-    const summaryDocument: DocumentIF = {
-      title: 'Summary',
-      filename: `${this.businessId} Summary - ${this.getCurrentDate}.pdf`,
-      link: `businesses/${this.businessId}/documents/summary`
-    }
-
-    await LegalServices.fetchDocument(summaryDocument).catch(error => {
-      // eslint-disable-next-line no-console
-      console.log('fetchDocument() error =', error)
-      this.downloadErrorDialog = true
-    })
-    this.setFetchingDataSpinner(false)
-  }
-
-  /** Direct to Digital Credentials. **/
-  viewAddDigitalCredentials (): void {
-    this.$router.push({ name: Routes.DIGITAL_CREDENTIALS })
-  }
```

Comment:
Does the Business Dashboard UI have an action to route to the DBC pages in this UI?

Reply:
Answered my own question -- yes, it does!

![image](https://github.com/user-attachments/assets/86ff0dfe-9512-451f-ac21-7b55fdab6ae7)